dinky-metadata/dinky-metadata-hive/pom.xml

```diff
@@ -56,14 +56,36 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-jdbc</artifactId>
+            <version>2.3.3</version>
+            <!--  If you use cdh hive comment out the previous line to open the next line and open the repositories below-->
+            <!--            <version>2.1.1-cdh6.3.2</version>-->
+            <scope>${scope.runtime}</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-service</artifactId>
+            <version>2.3.3</version>
+            <!--  If you use cdh hive comment out the previous line to open the next line and open the repositories below  -->
+            <!--            <version>2.1.1-cdh6.3.2</version>-->
+            <scope>${scope.runtime}</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.pentaho</groupId>
+                    <artifactId>pentaho-aggdesigner-algorithm</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
 
     </dependencies>
 
-    <repositories>
-        <repository>
-            <id>cloudera</id>
-            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
-        </repository>
-    </repositories>
+    <!--    <repositories>-->
+    <!--        <repository>-->
+    <!--            <id>cloudera</id>-->
+    <!--            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>-->
+    <!--        </repository>-->
+    <!--    </repositories>-->
```

Comment:
redundency

Reply:
> redundency

If it is the cdh version of hive, you need to open the comments here